pkg/apis/ceph.rook.io/v1/volume.go

```diff
@@ -48,3 +48,13 @@ func (src *ConfigFileVolumeSource) ToKubernetesVolumeSource() *corev1.VolumeSour
 
 	return dst
 }
+
+func (t *VolumeClaimTemplate) ToPVC() *corev1.PersistentVolumeClaim {
+	if t != nil {
+		return &corev1.PersistentVolumeClaim{
+			ObjectMeta: *t.ObjectMeta.DeepCopy(),
+			Spec:       *t.Spec.DeepCopy(),
+		}
+	}
+	return nil
```

Comment:
This is a bit of a nit: Idiomatically, Go prefers early-return methodologies to keep the largest body of the code flow at a non-indented level. I've heard this called "happy path on the left." This can be restructured slightly to match "idiomatic Go". In my experience, it is also widely-applicable to help keep C/Python/Bash/etc. code cleaner and easier to read.

I couldn't find a great link to the concepts, but there is a reddit thread discussing it [here](https://www.reddit.com/r/golang/comments/ywvx2f/idiomatic_use_of_assignment_in_conditional/).

```suggestion
	# handle returns as early as possible in short if statement
	if t == nil {
		return nil
	}
	# main body non-indented (on the left)
	return &corev1.PersistentVolumeClaim{
			ObjectMeta: *t.ObjectMeta.DeepCopy(),
			Spec:       *t.Spec.DeepCopy(),
		}
	}
```

Reply:
@BlaineEXE fixed, thanks for the suggestion. It does make sense and I think it's a good programming practice as it keeps the "happy" path easy to navigate when reading the code :)